js/gated-content/src/components/MediaPlayer.vue

```diff
@@ -38,6 +38,14 @@ export default {
     player() {
       return this.media.field_media_source === 'youtube' ? 'youtube' : 'vimeo';
```

Comment:
Videos were failing because `player` was returning `youtube_playlist`. This line should be:

```
      return this.media.field_media_source.startsWith('youtube') ? 'youtube' : 'vimeo';
```